### PR TITLE
Describe the known issue with localization in Angular 17.

### LINKED ISSUE #16816: Docs issue report from "getting-started/installation/angular.html"
### Origin URL

https://ckeditor.com/docs/ckeditor5/latest/getting-started/installation/angular.html

### Project version

42.0.2

### Is the information outdated? How?

I am experiencing issues with implementing internationalization in the editor for our Angular application. We are following the steps outlined here: https://ckeditor.com/docs/ckeditor5/latest/getting-started/installation/angular.html#localization.

Specifically, we have an issue with the following line:

import coreTranslations from 'ckeditor5/translations/sv.js';

It appears that no coreTranslations entity is being exported from that file.

### Is there something missing in the guide? What is it?

Additionally, I encountered discrepancies in the documentation regarding CSS imports. The documentation suggests importing the following file:

@import 'ckeditor5/ckeditor5.css';

However, this file does not exist. To make the editor work, I had to use a CDN link instead:

@import url("https://cdn.ckeditor.com/ckeditor5/42.0.2/ckeditor5.css");

It seems the documentation for Angular integration is not up to date.

### Is there anything else you would like to add?

Project details:

Angular: 17
Node: 18.19.1
@ckeditor/ckeditor5-angular: ^8.0.0
ckeditor5: ^42.0.2

### User agent

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Describe the known issue with localization in Angular 17. Closes #16816.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/integrations/angular.md [hunk 1]
```diff
@@ -690,6 +690,19 @@ export class AppComponent {
 
 For advanced usage see the {@link getting-started/setup/ui-language Setting the UI language} guide.
 
+<info-box warning>
+There is a known issue related to the localization in Angular 17. Read more in the [known issues section](#known-issues) below.
+</info-box>
+
+## Known issues
```
Comment (gorzelinski) [inline docs/getting-started/integrations/angular.md:697]: I was thinking about adding a subsection, but in our docs, there is rarely a subsection for the known issues.

File: docs/getting-started/integrations/angular.md [hunk 2]
```diff
@@ -690,6 +690,19 @@ export class AppComponent {
 
 For advanced usage see the {@link getting-started/setup/ui-language Setting the UI language} guide.
 
+<info-box warning>
+There is a known issue related to the localization in Angular 17. Read more in the [known issues section](#known-issues) below.
+</info-box>
```
Comment (gorzelinski) [inline docs/getting-started/integrations/angular.md:695]: I was wondering how to communicate this. I was thinking about three options:

1. Just a paragraph (It was easily missable).
2. Warning box at the start of the localization section (It was "scary").
3. Finally, I stick to the warning box at the end of the section.

File: docs/getting-started/integrations/angular.md (hunk 1 above)
Comment (gorzelinski) [reply docs/getting-started/integrations/angular.md:697]: And there is one known issue for now.

File: docs/getting-started/integrations/angular.md [hunk 3]
```diff
@@ -690,6 +690,19 @@ export class AppComponent {
...
+
+## Known issues
+
+The `moduleResolution` option of the TypeScript configuration determines the algorithm for finding and resolving modules from `node_modules`. In Angular 17, the option is set to `node` by default. This option prevents type declaration for editor translations from being correctly loaded. To fix it, you have several options:
+
+* You can set the `moduleResolution` option to `bundler`. It is the recommended setting in TypeScript 5.0+ for applications that use a bundler. And it is a recommended way of fixing this problem. You can check other solutions for lower TypeScript versions.
+* You can tell the TypeScript compiler to suppress the problem using the `// @ts-expect-error` comment above the imported translations.
+* You can update Angular to version 18, where the `moduleResolution` option is set to `bundler`  by default.
+* You can import translations directly from our CDN, like: `import ‘https://cdn.ckeditor.com/ckeditor5/42.0.2/translations/es.umd.js’;`. This way, the editor will load the translations automatically, so you do not need to pass them manually into the config.
```
Comment (pszczesniak) [inline docs/getting-started/integrations/angular.md:704]:
```suggestion
* You can import translations directly from our CDN, like: `import ‘https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/translations/es.umd.js’;`. This way, the editor will load the translations automatically, so you do not need to pass them manually into the config.
```

File: docs/getting-started/integrations/angular.md [hunk 4]
```diff
@@ -690,6 +690,19 @@ export class AppComponent {
...
 
+<info-box warning>
+There is a known issue related to the localization in Angular 17. Read more in the [known issues section](#known-issues) below.
+</info-box>
+
+## Known issues
+
+The `moduleResolution` option of the TypeScript configuration determines the algorithm for finding and resolving modules from `node_modules`. In Angular 17, the option is set to `node` by default. This option prevents type declaration for editor translations from being correctly loaded. To fix it, you have several options:
+
+* You can set the `moduleResolution` option to `bundler`. It is the recommended setting in TypeScript 5.0+ for applications that use a bundler. And it is a recommended way of fixing this problem. You can check other solutions for lower TypeScript versions.
```
Comment (pszczesniak) [inline docs/getting-started/integrations/angular.md:701]: > You can check other solutions for lower TypeScript versions.

You mean listed below, or other solutions for `moduleResolution`? 

This line doesn't tell exactly  what `other` solutions you meant - maybe would be good to specify it? :thinking:

File: docs/getting-started/integrations/angular.md (hunk 1 above)
Comment (pszczesniak) [reply docs/getting-started/integrations/angular.md:697]: I like the idea, and its present on some features docs so it's kind a template.

File: docs/getting-started/integrations/angular.md (hunk 3 above)
Comment (gorzelinski) [reply docs/getting-started/integrations/angular.md:704]: Good one - the link will be functional. Not just an example.

File: docs/getting-started/integrations/angular.md (hunk 4 above)
Comment (gorzelinski) [reply docs/getting-started/integrations/angular.md:701]: Maybe. If there is confusion, I can explicitly point to these below.

**Review by pszczesniak — APPROVED** [review]

**Review by Witoso — APPROVED** [review]

**Pull request merged**